.doc_gen/metadata/rds_metadata.yaml

```diff
@@ -0,0 +1,104 @@
+
+#zexi 0.4.0
+rds_CreateDBInstance:
+  title: Create an &RDSlong; instance and wait for it to be in an available state using an &AWS; SDK

```

Comment:
Shorten title to: Create an &RDS; instance using an &AWS; SDK

Reply:
fixed